include/kremlin/internal/types.h

```diff
@@ -49,19 +49,21 @@ typedef const char *Prims_string;
 
 /* The great static header headache. */
 
+#define IS_MSVC64 (defined(_MSC_VER) && defined(_M_X64) && !defined(__clang__))
+#define HAS_INT128 ((defined(__x86_64__) || defined(__x86_64) || \
+                     defined(__aarch64__) || \
+                     (defined(__powerpc64__) && defined(__LITTLE_ENDIAN__)) || \
+                     defined(__s390x__)) || \
+                    (defined(_MSC_VER) && !defined(_M_X64) && defined(__clang__)))
```

Comment:
Note that `__SIZEOF_INT128__` could be used to automatically add support to the platforms that aren't explicitly listed here but have compiler support for __int128.

Reply:
thanks, I've added that too in c3322295890570aeea0f240c0257910a907054de